pkg/debug/scripts/collect-info.sh

```diff
@@ -23,21 +23,24 @@ SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
 READ_LOGS_DEV=
 READ_LOGS_APP=
 TAR_WHOLE_SYS=
+OUT_LOGS_IN_JSON=
 
 usage()
 {
     echo "Usage: collect-info.sh [-v] [-h] [-a APPLICATION-UUID] [-d]"
     echo "       -h   show this help"
     echo "       -v   show the script version"
     echo ""
-    echo "Read-logs mode:"
+    echo "       -s   tar whole /sysfs"
+    echo ""
+    echo "Read-logs:"
     echo "       -d                   - read device logs only"
-    echo "       -s                   - tar whole /sysfs"
     echo "       -a APPLICATION-UUID  - read specified application logs only"
+    echo "       -j                   - output logs in json"
```

Comment:
IMO it should be inverted, if you provide the parameter then it should decode from json into plaintext, because we loose information, don't we?

Reply:
yes, json brackets mostly :)

Reply:
no one needs json in log output. simple fact. difficult to work, difficult to read. if you need json - here is the option for you "-j".

Reply:
and nested json is an absolute evil with escaped quotes and special symbols.

Reply:
What about `devID`, `image`, `eveVersion`?

It's not my fault that the log format is so evil, but loosing information is not good here!

Reply:
We don't lose anything, if you need more advanced log you rerun with '-j'.  What I'm saying is the default output was impossible to use anyway, you still have to convert it to text, which I did with another one-liner.

Are we talking about the same thing? This is only about post-processing, when all logs are on your machine already, so you need to sort them and unzip, which I did usually with the same collect-info.sh script.

Reply:
Further improvement can be if you provide fields which you would like to see in the log in text. That would be a good change, so if you can do that would be great.

Reply:
So look, when you got your tarball what you do? Can you describe your procedure?

I will describe mine. I don't care about the details, like eveVersion or devID or whatever, I need to see overall picture, I need to see the context, what caused an error. For that I need a clear text. I was doing the following:
```
./collect-info.sh > all-logs.json
cat all-logs.json | jq .......(another jq with a query to get clear text)
```

And the second `cat` was really annoying. So now the default behavior your get bare minimum to understand what has happened. Then, when  you need something advanced, you can call the ./collect-info.sh -j and get full json, because now you know what you are missing and what you are looking for. You see the point? From simple to more advance.

What's your procedure?

Reply:
>  but why is it in the collect-info.sh script then?

 because it simple:  when you get tarball with logs you don't need any extra tool, you call same collect-info.sh and get text output. 

> has something similar, doesn't he?

everyone in eve has it's own small secrets :)

Reply:
> because it is simple: when you get tarball with logs you don't need any extra tool, you call same collect-info.sh and get text output.

it is very confusing - I never have `collect-info.sh` on my machine as I don't run it there, but okay ...

Can you please add this to the usage?
How do I invoke it?
`collect-info.sh eve-info-v20-2024-ab-cd-ef-gh-ij.tar.gz` ?

Reply:
> it is very confusing - I never have collect-info.sh on my machine as I don't run it there, but okay ...

collect-info.sh is always with you, just in the eve-info-v20-2024.tar.gz folder.
so you receive your tarball, you untar them, you cd into the eve-info-v20-blahbla
and run script from the folder.

Reply:
> collect-info.sh is always with you, just in the eve-info-v20-2024.tar.gz archive. so you receive your tarball, you untar them, you cd into the eve-info-v20-blahbla and run script from the folder.

would be good to have this in the usage of `collect-info.sh`

Reply:
Yep, can describe this in usage more clearly.

Reply:
Done, please take a look.

Reply:
Thx!